labelImg.py

```diff
@@ -733,14 +766,10 @@ def button_state(self, item=None):
         if not item:  # If not selected Item, take the first one
             item = self.label_list.item(self.label_list.count() - 1)
 
-        difficult = self.diffc_button.isChecked()
+        difficult = self.difficult_checkbox.isChecked()
 
         try:
             shape = self.items_to_shapes[item]
-        except:
```

Comment:
Rationale: The second try/catch does not catch anything that the first did not catch anyway, so I integrated them into a single try/except block